### ISSUE TITLE
Bullet point style

### ISSUE THREAD

**Issue by benazir46** [matteobortolazzo/HtmlLabelPlugin#101]
Is it possible to style the bullet point from disk to circle or use an image instead?

**Comment by matteobortolazzo** [comment]
Which platform?

But the answer is no, it is handled natively

**Comment by benazir46** [comment]
Both iOS and Android. Do you have any how I can achieve this?

**Comment by matteobortolazzo** [comment]
Disks are a pretty specific and non-standard way to display lists.
Are you sure you are not using a custom font?

Because in iOS it is handled natively,
while in Android is handled by the library.

And has you can see [here](https://github.com/matteobortolazzo/HtmlLabelPlugin/blob/master/src/HtmlLabel/Android/ListBuilder.cs#L56) there are no disks

**Comment by benazir46** [comment]
oh yes, I'm using Roboto font. Here is how I'm using it:

            var label = new HtmlLabel
            {
                HorizontalOptions = LayoutOptions.StartAndExpand,
                VerticalOptions = LayoutOptions.StartAndExpand,
                Text = $"<div style=\"font-size: {fontSize}; text-align: left; font-family: Roboto; list-type:circle;\">{snippet.text.Replace("<p></p>", "&#10;")}</div>",
                HorizontalTextAlignment = TextAlignment.Start,
                FontFamily = "Roboto",
                FontSize = fontSize,
                AndroidLegacyMode = true,
                Padding = new Thickness(0),
                BackgroundColor = Color.Transparent,
            };

There is a difference on appearing lists in iOS and Android. Here in the picture you can see Android is adding extra space on the left, but iOS shows correctly. 

<img width="695" alt="list_difference" src="https://user-images.githubusercontent.com/21985993/83284647-ceb5af80-a1fe-11ea-84bc-0f72aa4c8368.png">

I'm also replacing \<p>\</p> with a new line code because without the conversion the label is adding one extra line.

**Comment by matteobortolazzo** [comment]
That's is handled by the OS itself, not much I can do, sorry

**Comment by benazir46** [comment]
> That's is handled by the OS itself, not much I can do, sorry

Could you update the indent on Android please? I think indent of 20 is creating the extra white space.

I was thinking why don't we create a separate control for the list with option to set images or font icons as bullet points and replace the list with the control.

**Comment by benazir46** [comment]
Without the legacy mode the list does not indent at all; it also does not show newline.

Ok, I'll create a PR if I come up with something.

**Comment by matteobortolazzo** [comment]
Oh ok, so there is a bug

**Comment by gemakwilia** [comment]
@matteobortolazzo 
My collegue (Daan) made a fix for IOS (link clicking) and will merge this (as promised). 
I created a fix for the listIndent on Android (which is also requested in this issue).

I added a property "AndroidListIndent" with default value 20 (as is).
This gives developers the opportunity to change it. 
Even negative value works excellent.

![image](https://user-images.githubusercontent.com/7520982/84144208-2a323980-aa58-11ea-81b8-17d995cdfb3f.png)

Result:
![image](https://user-images.githubusercontent.com/7520982/84144289-5057d980-aa58-11ea-97ee-3b4cd46be38d.png)

Instead of:
![image](https://user-images.githubusercontent.com/7520982/84144366-6cf41180-aa58-11ea-9d6b-ba79ee3a7e8c.png)

If it's OK by you we would like to add this as well.
